src/Utils/ErrorBoundary.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError() {
+
+    return { hasError: true };
+  }
+
+  static propTypes = {
```

Comment:
you can setup PropTypes same way you did for other components, let's stay consistent